extensions/kafka-streams/runtime/src/main/java/io/quarkus/kafka/streams/runtime/KafkaStreamsTopologyManager.java

```diff
@@ -141,8 +142,14 @@ public KafkaStreams getStreams() {
 
     private void waitForTopicsToBeCreated(Set<String> topicsToAwait, String bootstrapServersConfig)
             throws InterruptedException {
-        Map<String, Object> config = new HashMap<>();
+        final Map<String, Object> config = new HashMap<>();
         config.put(ConsumerConfig.BOOTSTRAP_SERVERS_CONFIG, bootstrapServersConfig);
```

Comment:
Now that I'm thinking about it, this should be this instead:
```suggestion
        config.put(AdminClientConfig.BOOTSTRAP_SERVERS_CONFIG, bootstrapServersConfig);
```

Reply:
You are right - that makes sense. I have updated the PR to use this suggestion.